README.md

```diff
@@ -1,77 +1,93 @@
 # voluspa
-> **// Ghost Proxy** / Proto-Warmind AI  
+
+> **// Ghost Proxy** / Proto-Warmind AI
 > _Destiny 2 Discord Bot_
 
 ![Voluspa Logo](images/voluspa/Voluspa_icon_100x133_black.png)
 
 ---
 
+### Local Docker for development
+
+A **Dockerfile** is provided that will also automagically setup a bot environment for local development, running, and debugging.
+
+#### Dockerfile Build
+
+```
+docker build . -t voluspa
+```
+
+#### Running the container
+
+```
+docker run -it voluspa:latest python3.8 ./voluspa.py
```

Comment:
I think it would be infinitely useful to include the Windows `docker run` command from the docker file here as well - it works great and is what I'm primarily using to develop.

Reply:
Oh, need to update the version also. What is the windows version of the command?

Reply:
All done :)